pattern-library/html/boilerplate-header+footer+content+nav.html

```diff
@@ -0,0 +1,113 @@
+        <div class="wrapper-view">
+
+            <!-- NOTE: wrapper elements are only needed when they need to manage the grid or larger view/page layout aspects (e.g. spanning the whole viewport) -->
+            <div class="wrapper-header-app">
+                <header class="header-app" role="banner">
+                    <!-- example: app logo/title -->
+                    <h1 class="header-app-title">
+                        <a class="logo" href="{app or company home url}">
+                            <img class="logo-img" src="{path/to/logo-img.svg}" alt="{app/site name}" />
+                            <!-- example: alternate app/site name (if not a graphical logo) -->
+                            <!-- {app/site name} -->
+                        </a>
+                    </h1>
+                    <!-- insert header content here -->
+
+                    <div class="wrapper-navigation">
+                        <nav class="nav-account" id="nav-account">
+                            <!-- NOTE: simplified user account nav example. This would be more complex in reality (with dropdowns, etc.) -->
+                            <a href="{url}">
+                                {username}
+                                <span class="sr-only">'s settings/profile</span>
+                            </a>
+                        </nav>
+                    </div>
+                </header>
+            </div>
+
+            <div class="wrapper-navigation">
+                <nav class="nav-app" id="nav-app">
+                    <a href="{url}">Course Home</a>
+                    <a href="{url}">Course Content</a>
+                    <a href="{url}">Discussions</a>
+                    <a href="{url}">Assignments</a>
+                    <a href="{url}">Assets &amp; Utilities</a>
+                    <a href="{url}">My Progress</a>
+                </nav>
+            </div>
+
+            <!-- NOTE: wrapper elements are only needed when they need to manage the grid or larger view/page layout aspects (e.g. spanning the whole viewport) -->
+            <div class="wrapper-content">
+                <main class="content-main" id="content-main">
+                    <!-- insert main content here -->
+                </main>
+
+                <aside class="content-aside" id="content-aside">
+                    <!-- insert aside/supplemental content here -->
+                </aside>
+            </div>
+
+            <!-- NOTE: wrapper elements are only needed when they need to manage the grid or larger view/page layout aspects (e.g. spanning the whole viewport) -->
+            <div class="wrapper-footer-app">
+                <footer class="footer-app" role="contentinfo">
+                    <!-- insert footer content here -->
+                </footer>
+            </div>
+        </div>
+
+        <!-- if needed, wrapper for all off-canvas DOM elements -->
+        <div class="wrapper-off-view">
```

Comment:
Since authors are almost always going to need to target this section to manage focus, it would probably be a good idea to give this div an ID attribute.